apps/DemoApp/DemoApp/ViewController.m

```diff
@@ -152,12 +170,12 @@ - (void)setupAVPlayerViewController:(AVPlayer *)player {
                                                      playerData:playerData
                                                       videoData:videoData
                                                        viewData: viewData];
-//    _imaListener = [[MuxImaListener alloc] initWithPlayerBinding:_playerBinding];
+    _imaListener = [[MuxImaListener alloc] initWithPlayerBinding:_playerBinding];
     [_avplayer play];
-
     [self addChildViewController:_avplayerController];
-    [self.view addSubview:_avplayerController.view];
-    _avplayerController.view.frame = self.view.frame;
+    _avplayerController.view.frame = self.view.bounds;
```

Comment:
Being a bit more explicit about where the view goes in the hierarchy and also calling the `didMoveToParentViewController` https://developer.apple.com/documentation/uikit/uiviewcontroller/1621405-didmovetoparentviewcontroller